### PR TITLE
Applications: nrf5340_audio: Remove bt ll acs nrf53

### PR DESCRIPTION
**Pull request by koffes**
Need to add doc before merging.

### CODE REVIEW

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst
Comment (koffes) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:1]: @greg-fer, should we delete this file/folder?

**Review by rick1082 — APPROVED** [review]
This is a huge PR, thanks so much for the great work from everyone who contributed to Audio Controller Subsystem!

**Review by greg-fer — CHANGES_REQUESTED** [review]
Shouldn't this module be removed in 2.9.0 at the earliest? The controller was replaced in 2.6.0, and we have the following guideline, provided by @tejlmand and @carlescufi and matching Zephyr:

> The Deprecated status is assigned to API that has gone through all maturity levels, but is being phased out. The deprecated API will be removed in one of future releases, **no earlier than two releases after the deprecation is announced** and only when the code has transitioned to not using the deprecated API. 

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/software_maturity.html#api-deprecation

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst
Comment (greg-fer) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:1]: Yes, but we need to provide redirects and ensure nothing breaks. I can take care of this after we confirm the module should be indeed removed at this point.

Comment (alexsven) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:1]: We have it confirmed that it can be removed now

**Review by alexsven — APPROVED** [review]

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst
Comment (greg-fer) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:1]: OK, let me work on redirects and doc removal later today.

**Comment by koffes** [comment]
@Frodevan : We need a separate PR to the test system which removes the use of the CONFIG_BT_LL_ACS_NRF53 flag.

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -149,7 +149,11 @@ Serial LTE modem
 nRF5340 Audio
 -------------
 
-|no_changes_yet_note|
+* Updated:
+
+  * The BT_LL_ACS_NRF53 controller has been removed.
+    The only supported controller for LE Audio is :ref:`ug_ble_controller_softdevice`.
+    This enables use of standard tools for building, configuring and DFU.
```
Comment (greg-fer) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:156]:
```suggestion
  * Removed the LE Audio controller for nRF5340 library.
    The only supported controller for LE Audio is :ref:`ug_ble_controller_softdevice`.
    This enables use of standard tools for building, configuring, and DFU.
```

**Review by greg-fer — APPROVED** [review]
@koffes , added commit with doc changes.
Please keep the double entry in the changelog, as we need a mention of the library removal in both areas, but kindly apply my edit.

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 1 above)
Comment (koffes) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]: Done

**Pull request merged**